### PR TITLE
Organize Storybook Components

### LINKED ISSUE #12551: Move "Getting Started" section to top in Storybook
**Describe the bug**
Getting Started section is at the bottom of the menu in storybook. It should be at the top.

**Steps to reproduce (REQUIRED)**
Steps to reproduce the behavior, libraries used with version number, and/or any setup information to easily reproduce:

1. Go to [Metamask storybook](https://metamask.github.io/metamask-storybook/index.html)
2. Scroll down menu to see "Getting Started" section at bottom

**Expected behavior**
Getting Started section should be at the top of the menu in storybook

We can use `storySort` to sort stories https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories

**Screenshots**
(Screen shot taken from #12424)

Screen Shot 2021-11-01 at 9 30 10 AM

### PR DESCRIPTION
**Pull request by EtDu**
This PR aims to classify and visually organise all storybook components.

Each component is sorted exactly how it appears in the project.

fixes #12551

### CODE REVIEW

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
This is looking good! I like the use of `'App/'`, `'UI/` and `'Pages'/` groupings. I think nesting any further may reduce discoverability? Also some of the groupings do not make complete sense to me e.g. `'UI/Modal/Actionable Message'`

I would suggest at least for this first run only having one level of groupings reflect the location of the stories in the app. This will help:

- Find stories more easily
- Understand how the code is organized 

Could we remove any nesting of groups further than `'App/'`, `'UI/` and `'Pages'/`?

So it would look something like

```
title: 'FOLDER_NAME/COMPONENT_NAME'
```

At least for this addition. We may be able to identify further nested groupings in future but for now I think this change will already make a huge difference!

**Review by brad-decker — APPROVED** [review]

**Comment by georgewrmarshall** [comment]
@EtDu also if we could fix this warning about duplicate story titles

Screen Shot 2021-10-26 at 11 28 54 AM

**Review by jj-crypto — APPROVED** [review]

**Review by jj-crypto — APPROVED** [review]

File: ui/pages/settings/settings.stories.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ import {
 import SettingsPage from './settings.component';
 
 export default {
-  title: 'SettingPage',
+  title: 'Pages/Send/Settings Page',
```
Comment (georgewrmarshall) [inline ui/pages/settings/settings.stories.js:22]: Should this be 
```suggestion
  title: 'Pages/Settings',
```

**Review by georgewrmarshall — COMMENTED** [review]
This looks excellent. Nice work! 💯  Just flagged one page but everything else looks great!

**Comment by georgewrmarshall** [comment]
I've also created this issue #12551 if you want to include it in this PR? Also fine as separate.

**Comment by EtDu** [comment]
> I've also created this issue #12551 if you want to include it in this PR? Also fine as separate.

good to go

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Nice work 💯

**Comment by georgewrmarshall** [comment]
To fix your lint issue just change `Base` story in `toggle-button.stories.js` to `DefaultStory`. Sorry I forgot I updated that one https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/ui/toggle-button/toggle-button.stories.js#L13

**Pull request merged**